src/openforms/authentication/contrib/org_oidc/tests/test_auth_procedure.py

```diff
@@ -298,11 +326,22 @@ def test_callback_url_creates_logged_in_djang_ouser(
             "given_name": "John",
             "family_name": "Doe",
             "arbitrary_employee_id_claim": "my_id_value",
+            "arbitrary_groups": ["registrators"],
         }
         mock_verify_token.return_value = user_claims
         mock_get_userinfo.return_value = user_claims
         mock_store_tokens.return_value = {"whatever": 1}
 
+        # setup user group
+        solo = OpenIDConnectConfig.get_solo()
+        group = GroupFactory(
+            name="registrators",
+            permissions=["of_authentication.can_register_client_submission"],
+        )
+        solo.save()
+        solo.default_groups.add(group)
```

Comment:
I'd swap these calls - the save triggers cache invalidation.

Reply:
Although this snippet isn't needed with the group in a fixture